ci/tests/plugin-compiler/test.sh

```diff
@@ -25,6 +25,7 @@ trap "$compose down" EXIT
 
 rm -fv testplugin/*.so || true
 docker run --rm -v `pwd`/testplugin:/plugin-source tykio/tyk-plugin-compiler:${tag} testplugin.so
+mv testplugin/testplugin*.so testplugin/testplugin.so
```

Comment:
Why we need this on master?
Plugin compiler for master generate output files with architecture and version in file name. 
Master gateway knows how to load such files. 
So this line should be only in release-4-lts branches I guess, which not yet support such plugin names?
And even release-4 (upcoming release-4.1) should support it just fine too... 

**UPDATE**
On the other thought I realized that in this test we load plugin via file, which kinda force you change file name in API definition every time. I raise this as a bug for this functionality here https://tyktech.atlassian.net/browse/TT-4354. 
Now this line does makes sense.